packages/fluentui/react-northstar/jest.config.js

```diff
@@ -1,13 +1,25 @@
+const path = require('path');
+const fs = require('fs');
+const { workspaceRoot } = require('@nrwl/devkit');
+const { pathsToModuleNameMapper } = require('ts-jest');
 const { createV0Config: commonConfig } = require('@fluentui/scripts-jest');
 
 const config = commonConfig({
-  name: 'react',
+  displayName: 'react-northstar',
   moduleNameMapper: {
     // Legacy aliases, they should not be used in new tests
```

Comment:
```suggestion
```

Remove a comment then?

Reply:
that comment is valid, hopefully nobody will use those aliases, but there is no lint rule for that and typescript will suggest you to use it so.... 🙃